mitmproxy/utils/bits.py

```diff
@@ -1,13 +0,0 @@
-def setbit(byte, offset, value):
```

Comment:
No other parts in the entire code base use these functions.
And we didn't have test for it either...